.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,6 +11,3 @@
 * Link to any supporting github issues or helpful documentation to add some context (e.g. stackoverflow). 
 * Use `closes #123`, if this PR closes a GitHub issue `#123`
 
-## NOTE
```

Comment:
It might be better to keep this commented out. If the `terraform/*` label isn't present then auto-format fails.

I believe this was added by @dylanbannon